doc/source/user_guide/launching_ansys_fluent.rst

```diff
@@ -193,4 +193,30 @@ For distributed parallel processing, you usually pass both parameters:
       version="3d",
       mode="solver",
       additional_arguments="-t16 -cnf=m1:8,m2:8",
-   )
\ No newline at end of file
+   )
+
+Using the PYFLUENT_ADDITIONAL_ARGUMENTS environment variable
+------------------------------------------------------------
+You can set additional arguments also using this environment variable.
+This can be useful for running on multiple platforms and allows you to keep
+a the same python script to use on the platforms.
+
+.. code:: bash
+
+   #!/bin/bash
+   #
+   # Activate your favorite Python environment
+   #
+   export AWP_ROOT232=/apps/ansys_inc/v232
+   . ./venv/bin/activate
+   #
+   # Set additional arguments
+   #
+   export PYFLUENT_ADDITIONAL_ARGUMENTS="-cnf=m1:8,m2:8 -t16 -peth.efa"
+   #
+   # Run a PyFluent script
+   #
+   python run.py
+
+Note: You should omit the additional_arguments option in the launch_fluent function as
+that variable is taken over the environmental variable.
```

Comment:
```suggestion
Note: You should omit the ``additional_arguments`` option in the ``launch_fluent`` function as
that option overrides what is set in the environment variable.
```

Reply:
Should it be the other way around? Env var overriding what is set in the `launch_fluent` arguments?

Reply:
I have updated the documentation